Helper/MetricsClient.php

```diff
@@ -381,4 +400,32 @@ public function postMetrics()
         }
         return null;
     }
+
+    /**
+     * Load data from Magento cache
+     *
+     * @param string $identifier
+     * @param bool $unserialize
+     * @return bool|mixed|string
+     */
+    protected function loadFromCache($identifier, $unserialize = true)
```

Comment:
@ethanwayda22 So, in general `unserialize` is not great to use. It's all internal data so it doesn't really matter, but it's against PSR-2 convention. Could you swap these to `json_encode` and `json_decode` instead?

(I know we have existing usages of `unserialize` in the project - they're grandfathered in and we should really clean it up someday)